documentation/api/guides/quickstart.md

```diff
@@ -1,47 +1,18 @@
-# Quickstart guide
+# API quickstart guide
 
 This is the quick start guide for setting up and running the API locally.
 
 ## Prerequisites
 
-Refer to the [general setup guide](/general/general_setup.md) for setting up the
-prerequisites. Refer to the 'API' column in the
-[requirement matrix](/general/general_setup.md#requirement-matrix) to know what
-you need to run this.
+Follow the [general setup guide](/general/general_setup.md) to set up `ov`.
 
 ## Starting up
 
-1. Ensure you download, install and set up all prerequisites. Ensure that the
-   Docker daemon is running.
-
-2. Clone the repository to your computer. Then switch to the cloned directory.
-   If you're planning to contribute, fork the repo and clone your fork instead.
-
-   ```{note}
-   We recommend cloning with the `--filter=blob:none` flag as it dramatically
-   reduces the filesize and download time by creating a "blobless clone".
-   You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
-   ```
-
-   ```bash
-   git clone --filter=blob:none https://github.com/WordPress/openverse.git # or your fork
-   cd openverse/
-   ```
-
-   If you followed the general setup guide and installed
-   [GitHub CLI](/general/general_setup.md#github-cli), you can clone more simply
-   using the `gh` command.
-
-   ```bash
-   gh repo clone WordPress/openverse -- --filter=blob:none  # or your fork
-   cd openverse/
-   ```
-
-3. Bring the ingestion server and API up, along with all their dependent
+1. Bring the ingestion server and API up, along with all their dependent
```

Comment:
```suggestion
1. Bring the ingestion server and API up, along with all of their dependent
```

Super nitpick

Reply:
This sentence is actually wrong anyway. It doesn't bring up their _dependent_ services, but their _dependencies_. It has the wrong direction. I'll fix this everywhere.